security-admin/src/main/java/org/apache/ranger/rest/ServiceREST.java

```diff
@@ -207,7 +207,7 @@ public ServiceREST() {
 	@POST
 	@Path("/definitions")
 	@Produces({ "application/json", "application/xml" })
-	@PreAuthorize("@rangerPreAuthSecurityHandler.isAPIAccessible(\"" + RangerAPIList.CREATE_SERVICE_DEF + "\")")
+	@PreAuthorize("@rangerPreAuthSecurityHandler.isServiceManagementAPIAccessible(\"" + RangerAPIList.CREATE_SERVICE_DEF + "\")")
```

Comment:
I really like your use of the middleware to scope privileges of the logged in user. Nice!